sample/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.secrets.gradle.plugin)
+}
+
+android {
+    namespace = "com.example"
```

Comment:
Suggestion: let's name this package into something more specific - if we ever publish, or want to filter by logs, or enable crash reports via crashlytics, etc it will come in handy. `com.afterpay.sample` ?

Reply:
- [ ] address this.